src/includes/traits/Plugin/UpdateUtils.php

```diff
@@ -116,9 +116,16 @@ protected function maybeCheckLatestProPackage()
         $product_api_response = wp_remote_post($product_api_url, ['body' => $product_api_input_vars]);
         $product_api_response = json_decode(wp_remote_retrieve_body($product_api_response));
 
+        $this->dismissMainNotice('pro_update_error'); // Clear any previous error notice in case a previous issue has been fixed.
+
+        if (!empty($product_api_response->error)) {
```

Comment:
@jaswsinc One thing that concerned me here was the scenario we fixed by implementing the proxy fallback, where an older version of cURL might be unable to connect to cometcache.com. I did a little research but couldn't figure out if in that scenario `$product_api_response->error` would be set. If it would, then we need to figure out how to work around that here, since in that case Comet Cache would be incorrectly reporting a username/password error and would never even try the proxy server below...